compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -109,6 +109,7 @@ class CircleOptimizer final
       UnrollUnidirSeqLSTM,
       XpSepActFromTransposeConv,
       RemoveGatherGuard,
+      CompressWeights
```

Comment:
As previous comment, I don't think we will add only one compression algorithm.
Please revise this to express the algorithm name.